l2/mkosi.build

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+ENV_YAML="$SRCDIR/l2/mkosi.extra/etc/flashbots/l2.yaml"
+
+VAULT_REF=$(mkosi-chroot yq -r .vault.git_reference < "$ENV_YAML")
+GOMPLATE_REF=$(mkosi-chroot yq -r .gomplate.git_reference < "$ENV_YAML")
+GCP_OPS_AGENT_REF=$(mkosi-chroot yq -r .gcp_ops_agent.git_reference < "$ENV_YAML")
+
+source scripts/make_git_package.sh
+
+# build gomplate
+
+make_git_package \
+    "gomplate" \
+    "${GOMPLATE_REF}" \
+    "https://github.com/hairyhenderson/gomplate" \
+    'go build -trimpath -ldflags "-s -w -X github.com/hairyhenderson/gomplate/v4/version=${GOMPLATE_REF} -buildid=" -o ./build/gomplate ./cmd/gomplate' \
+    "build/gomplate:/usr/bin/gomplate"
+chmod +x $DESTDIR/usr/bin/gomplate
+
+# build vault
+
+make_git_package \
+    "vault" \
+    "${VAULT_REF}" \
+    "https://github.com/hashicorp/vault.git" \
+    'go build -trimpath -ldflags "-s -w -X github.com/hashicorp/vault/version.Version=${VAULT_REF} -buildid=" -o ./bin/vault .' \
+    "bin/vault:/usr/bin/vault"
+chmod +x $DESTDIR/usr/bin/vault
+
+# build gcp ops agent
+
+cd "$BUILDROOT"
+IMPORT_PATH="github.com/GoogleCloudPlatform/ops-agent"
+BUILD_CMD="
+    # Install cmake 3.28 to fix build issues on Apple Silicon hosts
+    # See https://gitlab.kitware.com/cmake/cmake/-/issues/25562
```

Comment:
This won't work since fluent bit produces a different binary on MacOS vs Linux, we need to use the changes from commit fd3236acf652698a50123598ebdba016132aa745 in the l2-builder branch of the other repo instead.